src/components/LogIn.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom'
+import styles from '../../dist/panel.less';
+
+import ForgotPassword from './ForgotPassword';
+import SignUp from './SignUp';
+
+const LogIn = () => (
+	<div className="center-container">
+		<div className="panel">
+			<div className="panel-heading">
+				<h2>Log In</h2>
+			</div>
+			<div className="panel-body">
+				<form>
+					<input type="text" placeholder="Email" name="email" required />
```

Comment:
We could also make a global pure component for input so that there is always a label and input elements with same css style